packages/renderer/src/lib/dialogs/MessageBox.svelte

```diff
@@ -148,7 +150,7 @@ function getButtonType(b: boolean): ButtonType {
         {:else if type === 'question'}
           <Fa class="h-4 w-4" icon="{faCircleQuestion}" />
         {/if}
-        <h1 class="grow text-lg font-bold capitalize">{title}</h1>
+        <h1 id="{title}" class="grow text-lg font-bold capitalize">{title}</h1>
```

Comment:
What is this used for? Couldn't find a reference, maybe can be removed now.

Reply:
shouldn't be an id but an aria-label

Reply:
then `getByRole('heading', { name: '....' }`